compiler/src/main/resources/play_scala_security_extractors.mustache

```diff
@@ -8,6 +8,11 @@ import de.zalando.play.controllers.SwaggerSecurityExtractors._
 import {{import.name}}
 {{/for}}
 
+object SecurityExtractorsExecutionContext {
+    // TODO override with proper ExecutionContext instance
+    implicit val ec = scala.concurrent.ExecutionContext.global
```

Comment:
I was about to provide a separate execution context for token validation. This is I/O bound operation, so I think it makes sense to bulkhead it?